lib/config/rules/promise.js

```diff
@@ -1,13 +1,33 @@
 // see https://github.com/xjamundx/eslint-plugin-promise
 
 module.exports = {
+  // Promise Rules
+
   'promise/always-catch': 'off',
   // Ensure that each time a then() is applied to a promise, a catch() is applied as well. Exceptions are made if you are returning that promise.
   'promise/catch-or-return': 'warn',
+  // Avoid wrapping values in Promise.resolve or Promise.reject when not needed
+  'promise/no-return-wrap': 'error',
   // Ensure that inside a then() you make sure to return a new promise or value.
   'promise/always-return': 'error',
   // Enforce standard parameter names for Promise constructors.
   'promise/param-names': 'warn',
   // Ensure that Promise is included fresh in each file instead of relying on the existence of a native promise implementation.
   'promise/no-native': 'off',
+  // Avoid nested .then() or .catch() statements
+  'promise/no-nesting': 'warn',
+  // Avoid using promises inside of callbacks
+  'promise/no-promise-in-callback': 'warn',
```

Comment:
When working with any of the node API async methods, it's kind of hard to not do this, if you are using any other library that uses promises.